tests/integration/test_interaction.py

```diff
@@ -674,6 +674,44 @@ def test_do_attempt_feedback_is_updated(self):
         self.assertEqual(self._get_feedback().text, expected_feedback)
 
 
+class AssessmentGradingTest(
```

Comment:
@e-kolpakov, none other than I originally expected to have more tests.  Turns out it wasn't really necessary, so you're right, a class just for a single test seems overkill.